lib/generate-id.js

```diff
@@ -22,11 +22,41 @@ const findId = async ( model, length ) => {
 }
 
 /**
- * Generates a line id and a shabad id, using the database.
+   * Logs the generated id to the console
+   * @param {Number} count the count of id to generate
+   * @param {String} type shabad, line
+   */
+const logGeneratedID = async function ( count, type ) {
+  let i = 0
+
+  if ( type === 'shabad' ) {
+    for ( i = 0; i < count; i += 1 ) {
+      console.log( `${i + 1}. New Shabad ID: `.bold.magenta + `${await findId( Shabads, 3 )}`.rainbow )
```

Comment:
I know the old behaviour had "New Shabad ID" before each, but I was thinking it might be better to scrap this and just output the ID, purely because you may wish to do:

`npm run generate-id -- -t line -c 100 > line_ids.txt`

and then you've got a machine-parsable file by default.

Reply:
Performance issue: Each time `findId` is called, it queries the database and generates a set of IDs that are taken. You should refactor this to have the the DB only queried once.

Logic bug: New IDs that are generated are not filtered out, and therefore generated new IDs can have duplicates. You need to update the set, so maybe move the function and set into this function.